src/meta-client/src/client.rs

```diff
@@ -327,6 +328,28 @@ impl ClusterInfo for MetaClient {
 
         Ok(nodes)
     }
+
+    async fn list_region_stats(&self) -> Result<Vec<RegionStat>> {
+        let cluster_client = self.cluster_client()?;
+        let range_prefix = DatanodeStatKey::key_prefix_with_cluster_id(self.id.0);
+        let req = RangeRequest::new().with_prefix(range_prefix);
+        let mut datanode_stats = cluster_client
+            .range(req)
```

Comment:
Interesting... Use kv as API to fetch stats

Reply:
Yes, It's a in-memory key values store provided by metasrv.